tests/unit/amo/components/TestCategory.js

```diff
@@ -36,6 +38,22 @@ describe('Category', () => {
     );
   }
 
+  function mountRender(props = { loading: false }) {
+    const { store } = createStore();
```

Comment:
Instead of using `createStore` you should use the test helper `dispatchClientMetadata()` (which we should probably rename, sorry!) from `tests/unit/amo/helpers`. It dispatches things like lang and clientApp which should be available when using providers, links, etc.